### DISCUSSION TITLE
Is this orginiasation a paid one?

### DISCUSSION THREAD

**Discussion by albakhtari** [cstayyab/WALC#194] [General]
Is it a paid one or does it just use the free features?

**Comment by daffamumtaz2361** [comment]
Can you elaborate more? Pay to who? What features? Do you mean github or whatsapp?

**Comment by albakhtari** [comment]
Sorry for being ambiguous, I was talking about this github organization (WAClient), because I was looking at this:
![Screenshot_20220521_170849](https://user-images.githubusercontent.com/79008923/169650954-f6a58aea-c7b2-49a5-8e2b-4c500bb79d79.png)
